helpers/fuzzybool.go

```diff
@@ -0,0 +1,66 @@
+package helpers
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	// FuzzyBool resolves boolean sent as strings, integers, float64 or boolean, json unmarshal defaults to true
+	FuzzyBool bool
+)
+
+// UnmarshalJSON FuzzyBool
+func (sb *FuzzyBool) UnmarshalJSON(arg []byte) error {
+	if sb == nil {
+		return nil
+	}
+
+	tmpMap := make(map[string]interface{})
+	tmpArray := []byte(`{"val":`)
+	tmpArray = append(tmpArray, arg...)
+	tmpArray = append(tmpArray, byte('}'))
```

Comment:
Can't this be solved without this map trick?

With something like this:
``` golang
	var val interface{}
	err := json.Unmarshal(jsonRaw, &val)
	switch val.(type) {
	}
```

Reply:
hmm that actually seems to work now. It used to be interface{} was not an allowed type to allocate for json.unmarshal, when did that change?

Reply:
changed it, had missed that was allowed now. Also changed to use go-json instead.